scripts/travis.sh

```diff
@@ -26,8 +26,8 @@ if [[ $(git status -s) ]]; then
     exit 1
 fi
 
-PEGASUS_PKG="pegasus-tools-1.11.6-9f4e5ae-glibc2.12-release"
-PEGASUS_PKG_URL="https://github.com/XiaoMi/pegasus/releases/download/v1.11.6/pegasus-tools-1.11.6-9f4e5ae-glibc2.12-release.tar.gz"
+PEGASUS_PKG="pegasus-tools-1.13.SNAPSHOT-695b366-glibc2.17-release"
```

Comment:
I suggest adding some comments here to explain why update pegasus-tools to a SNAPSHOT version, and the time when the tools can be updated into a release version.